### PR TITLE
Adding detailed information on how to add secrets. 

### PR DESCRIPTION
**Pull request by rlam3**
Adding detailed information on how to add secrets.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -83,3 +83,4 @@ typings/
 
 # DynamoDB Local files
 .dynamodb/
+
```
Comment (Hephaest) [inline .gitignore:86]: A useless empty new line.

File: .github/workflows/schedule.yml [hunk 2]
```diff
@@ -1,15 +1,18 @@
...
+    # Reference: https://github.com/matchai/waka-box/issues/49
     - cron: "0 0 * * *"
 jobs:
   update-gist:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@master
       - name: Update gist
-        uses: matchai/waka-box@master
+        uses: rlam3/waka-box@master
```
Comment (Hephaest) [inline .github/workflows/schedule.yml:14]: DO NOT take your personal stuff into code review.

File: .github/workflows/schedule.yml [hunk 3]
```diff
@@ -1,15 +1,18 @@
...
         env:
           GH_TOKEN: ${{ secrets.GH_TOKEN }}
-          GIST_ID: 968220c97e8da1d047a9a480fa432e54
+          GIST_ID: 9e951e17fd4f1c62a32fdc8d7eff3bed
```
Comment (Hephaest) [inline .github/workflows/schedule.yml:17]: DO NOT take your personal stuff into code review.

File: README.md [hunk 4]
```diff
@@ -16,7 +16,10 @@
 1. Create a token with the `gist` scope and copy it. (https://github.com/settings/tokens/new)
 1. Create a WakaTime account (https://wakatime.com/signup)
 1. In your WakaTime profile settings (https://wakatime.com/settings/profile) ensure `Display coding activity publicly` and `Display languages, editors, operating systems publicly` are checked.
+1. Ensure that username and full name in your waka account is filled out as well.
```
Comment (Hephaest) [inline README.md:19]: Maybe this step is unnecessary since I didn't do any further work but still get the data.

**Review by Hephaest — CHANGES_REQUESTED** [review]
This PR needs more further modification.

File: .github/workflows/schedule.yml (hunk 3 above)
Comment (Leslie-23) [reply .github/workflows/schedule.yml:19]: Rightly said.

File: .github/workflows/schedule.yml [hunk 5]
```diff
@@ -11,5 +11,5 @@ jobs:
         uses: matchai/waka-box@master
         env:
           GH_TOKEN: ${{ secrets.GH_TOKEN }}
-          GIST_ID: 968220c97e8da1d047a9a480fa432e54
+          GIST_ID: f66b68a6bcd096a4f732e1e4c2e16323
```
Comment (Leslie-23) [inline .github/workflows/schedule.yml]: is this ID meant to be public

**Pull request closed without merging**